src/ringct/rctSigs.cpp

```diff
@@ -215,7 +206,8 @@ namespace rct {
               rv.II[i] = kLRki->ki;
             }
             else {
-              Hi = hashToPoint(pk[index][i]);
+              hash_to_p3(Hi_p3, pk[index][i]);
+              ge_p3_tobytes(Hi.bytes, &Hi_p3);
```

Comment:
Why not keep hashToPoint, whch uses hash_to_p3 internally ? Avoids these "inlining".

Reply:
if it isn't broke why fix it?

Reply:
Certainly could if desired. This is used infrequently.